src/plugins/intel_gpu/src/plugin/ops/parameter.cpp

```diff
@@ -174,107 +179,130 @@ static void CreateParameterOp(Program& p, const std::shared_ptr<ngraph::op::v0::
         break;
     }
 
-    if (ColorFormat::NV12 == preProcess.getColorFormat() && p.GetConfig().nv12_two_inputs) {
-        // for NV12, create two input layouts with reorder instead of one,
-        // and then would expect compound blob in inferRequest
-        if (Layout::NCHW != l &&
-            (Precision::I8 != ip || Precision::U8 != ip)) {
-            IE_THROW() << "Unsupported layout (" << l << ") or precision "
-                               << ip.name() << ") for NV12 input " + inputInfo->name();
+    bool is_convert_color_input = false;
+    for (auto& node : op->get_users()) {
+        is_convert_color_input |= ngraph::is_type<ngraph::op::v8::NV12toRGB>(node) ||
+                                  ngraph::is_type<ngraph::op::v8::NV12toBGR>(node);
+    }
+
+    if (is_convert_color_input) {
+        networkInputLayout.format = cldnn::format::byxf;
+        if (op->output(0).get_rt_info().count(ov::preprocess::TensorInfoMemoryType::get_type_info_static())) {
+            auto mem_type = std::dynamic_pointer_cast<ov::preprocess::TensorInfoMemoryType>(
+                            op->output(0).get_rt_info()[ov::preprocess::TensorInfoMemoryType::get_type_info_static()]);
+            if (mem_type->get() == GPUConfigParams::KEY_GPU_SURFACE) {
+                networkInputLayout.format = cldnn::format::nv12;
+            }
         }
-        int height = inputDims[2];
-        int width = inputDims[3];
-        std::vector<cldnn::primitive_id> reorders;
+        networkInputLayout.size = { TensorValue(inputDims[0]), TensorValue(inputDims[3]),
+                                    TensorValue(inputDims[2]), TensorValue(inputDims[1]) };
```

Comment:
Shouldn't it be like this for `byxf` format?
```
        networkInputLayout.size = { TensorValue(inputDims[0]), TensorValue(inputDims[2]),
                                    TensorValue(inputDims[3]), TensorValue(inputDims[1]) };
```

Reply:
No, its right
inputDims stores data in byxf format
Constructor cldnn::tensor requires bfxy format